tests/test_register.py

```diff
@@ -6,7 +6,46 @@
 from ingenialink.ethernet.register import EthernetRegister
 from ingenialink.exceptions import ILAccessError, ILValueError
 from ingenialink.register import REG_ACCESS, REG_DTYPE, REG_PHY, Register, dtypes_ranges
-from tests.virtual.test_virtual_network import connect_virtual_drive, stop_virtual_drive
+from ingenialink.virtual.network import VirtualNetwork
+from virtual_drive.core import VirtualDrive
+
+TEST_PORT = 82
+server = VirtualDrive(TEST_PORT)
+
+
+@pytest.fixture(autouse=True, scope="function")
+def stop_virtual_drive():
```

Comment:
Why do you execute this always?

Reply:
I just changed it [68465b5](https://github.com/ingeniamc/ingenialink-python/pull/400/commits/68465b5881efe413077c6cac483b1015ed2d7fd2)